.github/workflows/deploy.yml

```
@@ -0,0 +1,32 @@
+name: Build and Deploy ScratchJr Website
+
+on:
+    push:
+jobs:
+    build-and-deploy:
+        runs-on: ubuntu-latest
+        permissions:
+            id-token: write
+            contents: write
+        environment: ${{ github.ref == 'refs/heads/master' && 'production' || 'staging' }}
+        steps:
+        - uses: actions/checkout@v4
+        - name: Use Node 17x
+          uses: actions/setup-node@v3
+          with:
+            node-version: '17.x'
+        - name: Install Dependencies
+          run: npm install --legacy-peer-deps
```

Since this repo contains a `package-lock.json`, I think this should be:
```suggestion
          run: npm ci --legacy-peer-deps
```

---

good catch!